liquidity/liquidity.go

```diff
@@ -390,10 +424,36 @@ func cloneParameters(params Parameters) Parameters {
 	return paramCopy
 }
 
+// autoloop gets a set of suggested swaps and dispatches them automatically if
+// we have automated looping enabled.
+func (m *Manager) autoloop(ctx context.Context) error {
+	swaps, err := m.SuggestSwaps(ctx, true)
+	if err != nil {
+		return err
+	}
+
+	for _, swap := range swaps {
+		swap := swap
```

Comment:
nit: is this needed?

Reply:
yeah, reference to a range variable: https://medium.com/@betable/3-go-gotchas-590b8c014e0a
added a comment since it's not very clear.